README.md

```diff
@@ -156,6 +156,21 @@ sonar.pullrequest.base = target_branch_name (e.g master)
 pull-request. These properties indicate to sonar that a branch is being analyzed rather than a pull-request so no
 pull-request decoration will be executed.
 
+If you are scanning a GitHub pull request, you will also need to set the `sonar.scm.revision` argument.
+
+For example, using the official [SonarQube Scan](https://github.com/marketplace/actions/official-sonarqube-scan)
+on GitHub Actions:
+
+```yaml
+- name: SonarQube Scan
+  uses: sonarsource/sonarqube-scan-action@<action version>
+  args: >
+    -Dsonar.scm.revision=${{ github.event.pull_request.head.sha }}
```

Comment:
Hi,

Tried it, but it actually needs to be under "with:" section like this

```suggestion
  with:
    args: >
      -Dsonar.scm.revision=${{ github.event.pull_request.head.sha }}
```

![image](https://github.com/user-attachments/assets/3b49fca5-32d3-4c59-a892-6cae11f973cb)